justfile

```diff
@@ -10,7 +10,7 @@ test-print:
 build: 
 	cargo build --release
 
-run:
+r:
```

Comment:
You can define aliases, so you can call recipes with shorter names from the command line, while still keeping your justfiles legible. For example:

```
alias r := run
```

Reply:
got a little crazy with the aliases now